client/android/example/app/src/main/java/com/keepsafe/switchboard/example/NextActivityNormal.java

```diff
@@ -15,14 +15,13 @@
 */
 package com.keepsafe.switchboard.example;
 
-import android.app.Activity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
 
-public class NextActivityNormal extends Activity {
+public class NextActivityNormal extends AppCompatActivity {
```

Comment:
make this a normal activity so we can drop the dependency. As it's a sample for switchboard `Activity` should be enough